testscommon/factory/scResultPreProcessorFactoryStub.go

```diff
@@ -0,0 +1,21 @@
+package factory
+
+import (
+	"github.com/multiversx/mx-chain-go/process"
+	"github.com/multiversx/mx-chain-go/process/block/preprocess"
+	"github.com/multiversx/mx-chain-go/testscommon"
+)
+
+// SmartContractResultPreProcessorFactoryStub -
+type SmartContractResultPreProcessorFactoryStub struct {
```

Comment:
Duplicated with the one from `testscommon/factory/scResultPreProcessorFactoryMock.go`

Reply:
needed another one to not have an import cycle